runtime/util/ConfigurationPrinter.cpp

```diff
@@ -284,3 +288,89 @@ void printSortedConfigurationToFile(
   writer w = {file, nullptr};
   printConfigurationInternal(&w, subject, sort, false, &state);
 }
+
+void printMatchResult(
+    std::ostream &os, MatchLog *matchLog, size_t logSize,
+    std::string &definitionPath) {
+  char subjectFilename[15] = "subject_XXXXXX";
+  int sf = mkstemp(subjectFilename);
+  FILE *subject = fdopen(sf, "w");
+
+  char patternFilename[15] = "pattern_XXXXXX";
+  int pf = mkstemp(patternFilename);
+  FILE *pattern = fdopen(pf, "w");
+
+  if (logSize > 0) {
+    for (int i = 0; i < logSize; i++) {
+      if (matchLog[i].kind == MatchLog::SUCCESS)
+        os << "Match succeeds\n";
```

Comment:
Very minor, but let's add braces to `if` statements. Unfortunately clang format won't catch this for us